analytic_tag_dimension/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Analytic Tag Dimension
+======================
+
+This module allows to group Analytic Tags on Dimensions.
+
+Dimensions are created as custom field, then you can group by Dimensions on Analytic Entries Report.
+
+When you set Tags on Analytic Entries each custom fields for dimensions are updated with its Tag. One Tag is only allow on one Dimension, and you can not set more than one Tag from same Dimensions on Analytic Entry.
+
+
+Configuration
+=============
+
+To configure this module, you need to go to /Accounting/Configuration/Analytic Accounting/Analytic Accounts Dimensions to create new Analytic Dimensions.
```

Comment:
please use enumerated list for the procedure (with "#. "). It is easier to follow